civihr_default_theme/scss/base/_variables.scss

```diff
@@ -116,7 +116,8 @@ $nav-tabs-justified-active-link-border-color: $light !default;
 // Media queries breakpoints
 // ------------------------------------------------------------------------------------------------------
 
-$screen-sm-max:               767px !default;
+$screen-sm-max:                   767px !default;
+$crm-menu-responsive-breakpoint:  1250px;
```

Comment:
It's `crm_`, not `crm-`

Reply:
Its `crm-` only, I have checked other instances, 
For example: https://github.com/compucorp/civihr-employee-portal-theme/blob/d0c40f87036964df1f9b2d013de26b05415c46c0/civihr_default_theme/scss/base/_variables.scss#L126-L126